src/Build/Evaluation/IntrinsicFunctions.cs

```diff
@@ -506,6 +506,11 @@ internal static string GetTargetPlatformVersion(string tfm, int versionPartCount
             return NuGetFramework.Value.GetTargetPlatformVersion(tfm, versionPartCount);
         }
 
+        internal static bool AreFeaturesEnabled(string wave)
+        {
+            return string.IsNullOrEmpty(wave) ? false : ChangeWaves.AreFeaturesEnabled(wave);
```

Comment:
Important to answer, maybe not to change.

This means that if you don't set it to anything, no feature is enabled? I would've though the opposite would be the default.

Reply:
This is for the argument to the function, not the environment variable itself. 

If you pass an empty value to the intrinsic function, say `$([MSBuild]::AreFeaturesEnabled('$(SomeProperty)')` where `SomeProperty` resolves to an empty string, the feature will be as disabled.